### PR TITLE
Feature/readme.md

### PR DESCRIPTION
**Pull request by jgiwinski**
## What does this PR do?
 - Updates readme.md file with description of How To Use app.
 
## How should it be tested?
 - NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
-# Refactor-Tractor
+# FitLit (Refactor-Tractor)
+
+## Introduction
+_FitLit_ is a new app to help anyone begin and maintain their fitness journey.  By logging one's daily hydration, activity, and sleep habits users can keep track of their progress on the path to healthy living.
```
Comment (knees4bees) [inline README.md:4]: Short and clear!

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+## How to use:
++ When the user first loads the page they will see a nav bar with their profile photo and several icons. Each of these icons can be clicked on to display the relevant data. Moving to the right side of the nav bar the user will notice several input fields. This is where the user can input a date and their daily stats for hydration, activity, and sleep. The user will be prompted with the type of data required for each field. If a field isn't properly filled out then an error message will display telling the user how to correct their input.
```
Comment (knees4bees) [inline README.md:7]: Nice fleshing out of this description.

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+
+The Activity section displays the user's daily stats for the number of steps walked, total flights of stairs climbed, and total number of active minutes. These stats are stacked on top of the average number of steps walked, total flights of stairs climbed, and total number of active minutes for all participants who use the app.
+Under the user's daily stats there is a section that shows the total miles walked and an additional section that displays three charts. These charts show weekly data for number of steps walked, total flights of stairs climbed, and total number of active minutes.
+
+In the Sleep section of the page the user will be shown the total number of hours they slept the night before, the quality of sleep (ranked 1-5) the user had, and the average quality of sleep they regularly get.
+Under those stats there are two charts that display weekly data for the total number of hours slept and the sleep quality of each night.
+
+
+<SCREENSHOT OF ALL THREE DATA DISPLAY SECTIONS>
```
Comment (knees4bees) [inline README.md:20]: Yep, we'll need to remember to add some screenshots

**Review by knees4bees — APPROVED** [review]

**Comment by rdtho2525** [comment]
Looks great!

**Pull request merged**